src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -226,6 +228,7 @@
     public static final int ETH_SRC_OFFSET = 48;
     public static final int INTERNAL_ETH_SRC_OFFSET = 448;
     public static final int MAC_ADDRESS_SIZE_IN_BITS = 48;
+    public static final String LLDP_MAC = "01:80:c2:00:00:0e";
```

Comment:
LLDP_MAC was removed in #3169 
This mac is unused now because we decided to remove it from all LLDP matches.
So please do not add this mac back

Reply:
Thanks, I have found this issue too. Working on it now.